Sources/GRPCCodeGen/Internal/Translator/ClientCodeTranslator.swift

```diff
@@ -264,6 +275,18 @@ extension ClientCodeTranslator {
     )
   }
 
+  private func callOptionsParameter(
```

Comment:
All this function does is call the `init` of `ParameterDescription` and this function is only ever used in one place. Having a function for this makes the code _less_ readable as there is more indirection.

It'd be better to just create the `ParameterDescription` inline in `makeParameters`.